_episodes/09-basic-visualizations.md

```diff
@@ -18,13 +18,11 @@ against the appropriate stations.
 # ?glatos::abacus_plot
```

Comment:
title and question should be changed to glatos

Reply:
Fixed